Makefile

```diff
@@ -41,10 +41,20 @@ create-db:
 
 apply-migrations: ensure-venv
 	@echo "--> Applying migrations"
-	sentry upgrade
+	sentry upgrade --noinput
+
+create-user:
```

Comment:
I'll give this another quick pass after you rebase/remerge this, things like these would become a part of `./scripts/do.sh $@`.